Library/Formula/httpie.rb

```diff
@@ -1,11 +1,11 @@
 class Httpie < Formula
   desc "User-friendly cURL replacement (command-line HTTP client)"
-  homepage "https://github.com/jakubroztocil/httpie"
-  url "https://github.com/jakubroztocil/httpie/archive/0.9.2.tar.gz"
-  sha256 "1f0b0b6563de25bad0ed190d62f1130573124201382824756d015b1422a234cc"
+  homepage "http://httpie.org/"
```

Comment:
This seems to just redirect back to `https://github.com/jkbrzt/httpie` at the moment. Is the plan to change that?